### PR TITLE
Improve vscode documentation

### PR DESCRIPTION
**Pull request by anuruddhal**
## Purpose
- Improve vscode documentation
- Fixes https://github.com/ballerina-platform/ballerina-dev-website/issues/8986

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: swan-lake/vs-code-extension/notebooks.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ You can create Ballerina notebooks in the VS Code editor (by adding the `.balnot
 
 ## Execute the code
 
-Click the **Run** button available at the left-hand side of the cell or press the `shift` + `enter` keys to execute the Notebook cells. 
+Click the **Run** button available at the left-hand side of the cell to execute the Notebook cells.
+Alternatively, you can use the `Shift` + `enter` shortcut methods to execute the cells.
```
Comment (sm1990) [inline swan-lake/vs-code-extension/notebooks.md:19]:
```suggestion
Alternatively, you can use the `Shift` + `enter` shortcut method to execute the cells.
```

**Comment by sm1990** [comment]
LGTM

File: swan-lake/vs-code-extension/write-the-code/code-navigation.md [hunk 2]
```diff
@@ -1,38 +1,43 @@
...
 
 The [Go to definition](https://code.visualstudio.com/docs/editor/editingevolved#_go-to-definition) feature navigates you to the definition of a particular symbol. For example, when you invoke the `Go To Definition` option on a function call expression, it navigates you to the definition of the function.
+To go to a symbol's definition, press `F12`. On Windows and Linux, hold `Ctrl`, or on Mac, hold `⌘` and hover over the symbol to see a preview of its declaration.
 
 <img src="/learn/images/vs-code-extension/edit-the-code/code-navigation/go-to-definition.gif" class="cInlineImage-full"/>
 
 ## Go to symbol
 
-The [Go to symbol](https://code.visualstudio.com/docs/editor/editingevolved#_go-to-symbol) feature allows you to navigate symbols inside a file.
+The [Go to symbol](https://code.visualstudio.com/docs/editor/editingevolved#_go-to-symbol) feature allows you to navigate symbols inside a file. You can navigate symbols within a file by pressing `⇧` + `⌘ `+`O` on a Mac or `Ctrl` + `Shift` + `O` on Linux and Windows.
```
[the remark this answers is not in the document]
Comment (anuruddhal) [reply swan-lake/vs-code-extension/write-the-code/code-navigation.md]: Used `Shift` for windows/linux and `shift` for mac.

File: swan-lake/vs-code-extension/write-the-code/code-navigation.md [hunk 3]
```diff
@@ -2,37 +2,48 @@
 title: Code navigation
 description: The Ballerina VS Code extension provides code navigation to simplify the developing process of your code. 
 keywords: ballerina, vs code extension, code navigation
-intro: The Ballerina VS Code extension provides code actions to simplify the developing process of your code. 
+intro: The Ballerina VS Code extension provides support for code navigation.
 ---
 
-The Ballerina VSCode extension provides support for [code navigation](https://code.visualstudio.com/docs/editor/editingevolved). Different kinds of supported navigations are described in the following sections.
+The Ballerina VS Code extension provides support for [code navigation](https://code.visualstudio.com/docs/editor/editingevolved). Different kinds of supported navigations are described in the following sections.
```
Comment (MaryamZi) [inline swan-lake/vs-code-extension/write-the-code/code-navigation.md:8]:
```suggestion
The Ballerina VS Code extension provides support for [code navigation](https://code.visualstudio.com/docs/editor/editingevolved). The different kinds of supported navigations are described in the following sections.
```

File: swan-lake/vs-code-extension/write-the-code/code-navigation.md [hunk 4]
```diff
@@ -2,37 +2,48 @@
...
 
+To go to a symbol's definition, press `F12`. Hold `Ctrl` on Windows and Linux or `⌘` on macOS and hover over the symbol to see a preview of its declaration.
+
 <img src="/learn/images/vs-code-extension/edit-the-code/code-navigation/go-to-definition.gif" class="cInlineImage-full"/>
 
 ## Go to symbol
 
 The [Go to symbol](https://code.visualstudio.com/docs/editor/editingevolved#_go-to-symbol) feature allows you to navigate symbols inside a file.
 
+You can navigate symbols within a file by pressing `Ctrl` + `Shift` + `O` or `shift` + `⌘ `+`O` on a Mac.
```
Comment (MaryamZi) [inline swan-lake/vs-code-extension/write-the-code/code-navigation.md:22]: Doesn't say Windows or Limux for the first one.

Comment (MaryamZi) [reply swan-lake/vs-code-extension/write-the-code/code-navigation.md:22]: Shall we be consistent with macOS vs a Mac?

File: swan-lake/vs-code-extension/write-the-code/code-navigation.md [hunk 5]
```diff
@@ -2,37 +2,48 @@
...
+To go to a symbol's definition, press `F12`. Hold `Ctrl` on Windows and Linux or `⌘` on macOS and hover over the symbol to see a preview of its declaration.
+
 <img src="/learn/images/vs-code-extension/edit-the-code/code-navigation/go-to-definition.gif" class="cInlineImage-full"/>
 
 ## Go to symbol
 
 The [Go to symbol](https://code.visualstudio.com/docs/editor/editingevolved#_go-to-symbol) feature allows you to navigate symbols inside a file.
 
+You can navigate symbols within a file by pressing `Ctrl` + `Shift` + `O` or `shift` + `⌘ `+`O` on a Mac. 
+By typing `:` the symbols will be grouped by category.
```
Comment (MaryamZi) [inline swan-lake/vs-code-extension/write-the-code/code-navigation.md:23]:
```suggestion
The symbols can be grouped by category by typing `:`.
```

File: swan-lake/vs-code-extension/write-the-code/commands.md [hunk 6]
```diff
@@ -7,18 +7,18 @@ intro: You can use the commands below when editing Ballerina code via the VS Cod
 
  Click **View** in the menu bar of the editor, and then, click **Command Palette** to access them.
 
->**Tip:** Alternatively, you can use the `⌘ + ↑ + P` shortcut methods on Mac and `Ctrl + Shift + P` on Windows and Linux to open the **Command Palette**.
+>**Tip:** Alternatively, you can use the `⌘ + shift + P` shortcut methods on Mac and `Ctrl + Shift + P` on Windows and Linux to open the **Command Palette**.
```
Comment (MaryamZi) [inline swan-lake/vs-code-extension/write-the-code/commands.md:10]: Order not changed.

File: swan-lake/vs-code-extension/notebooks.md [hunk 7]
```diff
@@ -15,7 +15,8 @@ You can create Ballerina notebooks in the VS Code editor (by adding the `.balnot
 
 ## Execute the code
 
-Click the **Run** button available at the left-hand side of the cell or press the `shift` + `enter` keys to execute the Notebook cells. 
+Click the **Run** button available on the left-hand side of the cell to execute the Notebook cells.
+Alternatively, you can use the `shift` + `enter` keyboard shortcut to execute the cells.
```
Comment (MaryamZi) [inline swan-lake/vs-code-extension/notebooks.md:19]: Can this be
```suggestion
Alternatively, you can use the `Shift` + `Enter` keyboard shortcut to execute the cells.
```

File: swan-lake/vs-code-extension/write-the-code/code-navigation.md [hunk 8]
```diff
@@ -1,38 +1,43 @@
...
 
 This feature allows you to [peek](https://code.visualstudio.com/docs/editor/editingevolved#_peek) the definition or references of a given symbol.
+On Windows and Linux, press `Ctrl` + `Shift` + `F10`, or on Mac, press `⇧` + `F12` to open the peek view.
 
 <img src="/learn/images/vs-code-extension/edit-the-code/code-navigation/peek-definition.png" class="cInlineImage-full"/>
 
 ## Rename symbols
 
 This feature allows you to rename symbols by renaming all the references of the particular symbol.
+Press `F2`, type the new desired name, and then press `Enter`. This will rename all usages of the symbol across files.
```
[the remark this answers is not in the document]
Comment (MaryamZi) [reply swan-lake/vs-code-extension/write-the-code/code-navigation.md]: Ping on this too.

File: swan-lake/vs-code-extension/write-the-code/commands.md [hunk 9]
```diff
@@ -7,18 +7,18 @@ intro: You can use the commands below when editing Ballerina code via the VS Cod
 
  Click **View** in the menu bar of the editor, and then, click **Command Palette** to access them.
 
->**Tip:** Alternatively, you can use the `⌘ + ↑ + P` shortcut methods on Mac and `Ctrl + Shift + P` on Windows and Linux to open the **Command Palette**.
+>**Tip:** Alternatively, you can use the `Ctrl` + `Shift` + `P` on Windows and Linux `⌘` + `sgithift` + `P` shortcut methods on Mac to open the **Command Palette**.
```
Comment (MaryamZi) [inline swan-lake/vs-code-extension/write-the-code/commands.md:10]:
```suggestion
>**Tip:** Alternatively, you can use `Ctrl` + `Shift` + `P` on Windows and Linux `⌘` + `shift` + `P` on Mac to open the **Command Palette**.
```

File: swan-lake/vs-code-extension/write-the-code/commands.md [hunk 10]
```diff
@@ -7,18 +7,18 @@ intro: You can use the commands below when editing Ballerina code via the VS Cod
 
  Click **View** in the menu bar of the editor, and then, click **Command Palette** to access them.
 
->**Tip:** Alternatively, you can use the `⌘ + ↑ + P` shortcut methods on Mac and `Ctrl + Shift + P` on Windows and Linux to open the **Command Palette**.
+>**Tip:** Alternatively, you can use the `Ctrl` + `Shift` + `P` on Windows and Linux or `⌘` + `shift` + `P` shortcut methods on Mac to open the **Command Palette**.
```
Comment (MaryamZi) [inline swan-lake/vs-code-extension/write-the-code/commands.md:10]:
```suggestion
>**Tip:** Alternatively, you can use the `Ctrl` + `Shift` + `P` on Windows and Linux or `⌘` + `shift` + `P` on Mac to open the **Command Palette**.
```

File: swan-lake/vs-code-extension/write-the-code/commands.md [hunk 11]
```diff
@@ -7,18 +7,18 @@ intro: You can use the commands below when editing Ballerina code via the VS Cod
 
  Click **View** in the menu bar of the editor, and then, click **Command Palette** to access them.
 
->**Tip:** Alternatively, you can use the `⌘ + ↑ + P` shortcut methods on Mac and `Ctrl + Shift + P` on Windows and Linux to open the **Command Palette**.
+>**Tip:** Alternatively, you can use the `Ctrl` + `Shift` + `P` on Windows and Linux or `⌘` + `shift` + `P` on Mac to open the **Command Palette**.
```
Comment (MaryamZi) [inline swan-lake/vs-code-extension/write-the-code/commands.md:10]:
```suggestion
>**Tip:** Alternatively, you can press `Ctrl` + `Shift` + `P` on Windows and Linux or `⌘` + `shift` + `P` on Mac to open the **Command Palette**.
```

File: swan-lake/vs-code-extension/write-the-code/commands.md [hunk 12]
```diff
@@ -7,18 +7,18 @@ intro: You can use the commands below when editing Ballerina code via the VS Cod
 
  Click **View** in the menu bar of the editor, and then, click **Command Palette** to access them.
 
->**Tip:** Alternatively, you can use the `⌘ + ↑ + P` shortcut methods on Mac and `Ctrl + Shift + P` on Windows and Linux to open the **Command Palette**.
+>**Tip:** Alternatively, you can press the `Ctrl` + `Shift` + `P` on Windows and Linux or `⌘` + `shift` + `P` on Mac to open the **Command Palette**.
```
Comment (MaryamZi) [inline swan-lake/vs-code-extension/write-the-code/commands.md:10]:
```suggestion
>**Tip:** Alternatively, you can press `Ctrl` + `Shift` + `P` on Windows and Linux or `⌘` + `shift` + `P` on Mac to open the **Command Palette**.
```

**Review by MaryamZi — APPROVED** [review]

**Review by sm1990 — APPROVED** [review]

**Pull request merged**